lib/std/start.zig

```diff
@@ -102,12 +102,11 @@ fn main2() callconv(.C) c_int {
     return 0;
 }
 
-fn _start2() callconv(.C) noreturn {
+fn _start2() callconv(.withStackAlign(.c, 1)) noreturn {
```

Comment:
Do decl literals allow:

```suggestion
fn _start2() callconv(.winapi.withStackAlign(1)) noreturn {
```

? (This doesn't matter much at all; just curious.)

Reply:
Nope, not allowed. Given `foo.bar()` with result type `T`, we can't forward any meaningful result type to `foo`, so supporting expressions of this form would have to be a special case.